feat(VMaskInput): Multiple and dynamic mask resolution (#22501)
vuetifyjs/vuetify: packages/docs/src/pages/en/components/mask-inputs.md

<ExamplesExample file="v-mask-input/custom-token" />

#### Multi-Mask with Dynamic Resolution

> You can provide an array of masks to allow dynamic mask selection based on user input. The component automatically selects the best matching mask as the user types:

Reviewer: ```suggestion
Provide an array of masks to enable dynamic selection based on user input. The component selects the best matching mask as the user types:
```
